bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/ElectionType.cs

```diff
@@ -9,34 +9,34 @@ public enum ElectionType
         /// <summary>
         /// an unknown value
         /// </summary>
-        Unknown = 0,
+        unknown = 0,
```

Comment:
for all of these, this is an option to consider...especially since this is a compatibility problem:

```suggestion
        Unknown = 0,
        
        [Obsolete("Not in use anymore")]
        unknown = 0,
```